app/main/views.py

```diff
@@ -99,6 +165,43 @@ def get_service(service_id):
     return jsonify(services=jsonify_service(service))
 
 
+@main.route('/services-archive/<int:service_archive_id>', methods=['GET'])
```

Comment:
As we addressing a domain object - the service archive - that forms the root part of the URL, then it's unique id for a canonical row, or a query string for filtering.

We're not hitting the services table, hence not using that in the URL.

Happy to chat more, will not change it for now though.